### DISCUSSION TITLE
Error on v1.0 with Linux cmake build

### DISCUSSION THREAD

**Discussion by club660** [vincentjzy/OpenCorr#10] [Q&A] [answered]
make a OpenCorr v1.0 build with following error on 'oc_point'

`error: ‘sqrt’ was not declared in this scope`
   `42 |                 return sqrt(x * x + y * y);`
     `   |                        ^~~~`

Update:
Add `#include <math.h>` at the beginning solve this problem.

**Comment by vincentjzy** [comment, marked as answer]
Thank you very much for testing OpenCorr in Linux and giving the suggestion. We have added the line into oc_point.cpp.
